Answers.md

```diff
@@ -1,3 +1,15 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+## 1. Containers are faster and use less resources than virtual machines.
```

Comment:
How are they faster than virtual machines? What is it about containers that allows them to achieve this?